extensions/ql-vscode/test/vscode-tests/no-workspace/contextual/astBuilder.test.ts

```diff
@@ -52,11 +53,12 @@ describe("AstBuilder", () => {
     const astBuilder = createAstBuilder();
     const roots = await astBuilder.getRoots();
 
+    const bqrsPath = path.normalize("/a/b/c/results.bqrs");
```

Comment:
Any understanding of why this test had to be changed?

Reply:
I think that change went into the base PR, but the reason for the change was that previously, the test code passed in `"/a/b/c"` as the path to the BQRS file itself, which got passed through the implementation verbatim and came out the other end with forward slashes intact. With my changes, `"/a/b/c"` is treated as the query output directory, which goes through some `path` calls (`join`, etc.) before the BQRS path comes out the other end with platform-specific separators.